src/bentoml/_internal/marshal/dispatcher.py

```diff
@@ -234,8 +234,6 @@ async def train_optimizer(
                     # fake wait as 0 for training requests
                     info.enqueue_time = now
                 self._loop.create_task(self.outbound_call(inputs_info))
-        except asyncio.CancelledError:
-            return
```

Comment:
Huh, did this not get replaced by `raise` earlier? If you don't re-raise it manually it'll get caught by the catch-all below, right?